app/Traits/HasReferralLink.php

```diff
@@ -6,6 +6,12 @@ trait HasReferralLink
 {
     public function get_referral_link(): string
     {
+        // TODO: Add a proper referral code
+        $code = 'TODO'; // $this->promoter;
+        // FIXME: This code is currently using the home page
```

Comment:
My guess on this would be to have referrals either point to the register route, which upon "noticing a «referral» visit" would act accordingly, or a "blank" route (i.e., which shows no content) that processes the fact that the request is from a referral link and then forwards the user to the registration form. Ideas? I have already implemented the first option if I'm not mistaken.

Reply:
You have, yeah. The first approach works, I don't remember any detail that would make the second one needed.

Reply:
Guess that is that then. Will mark this as resolved.